readthedocs/templates/projects/project_version_list.html

```diff
@@ -134,8 +135,19 @@ <h1>{% trans "Activate a version" %}</h1>
           </div>
         {% endif %}
 
+
+
       </div>
     </div>
+
+    {% if inactive_versions|length >= max_inactive_versions %}
+      <p>
+        {% blocktrans trimmed %}
+          Only the first {{ max_inactive_versions }} inactive versions were shown. Please <a href="#version-filter">filter the list</a>.
```

Comment:
This will involve an extra query as I outlined https://github.com/readthedocs/readthedocs.org/pull/6276#discussion_r343881508 but I suppose the cost isn't too high. I can add it.